### PR TITLE
Eenable locale for Serbian Latin Serbia language

### PR DESCRIPTION
**Pull request by matmsa27**
Hi everyone, I open this PR to have available the Serbian Latin Language (`sr_RS@latin`) on the UReport. This locale on transifex has 100% translated.

### CODE REVIEW

File: ureport/settings_common.py [hunk 1]
```diff
@@ -83,6 +83,7 @@
     ("ro", "Romanian"),
     ("vi", "Vietnamese"),
     ("sr-latn", "Latin Serbian"),
+    ("sr-rs@latin", "Serbian (Latin, Serbia)"),
```
Comment (norkans7) [inline ureport/settings_common.py]: What is the previous one("sr-latn") for?

Comment (norkans7) [reply ureport/settings_common.py]: I think we only need one of them not both

Comment (matmsa27) [reply ureport/settings_common.py]: There are the two folders on locale for Serbia Language, the sr-rs@latin has 100% translated on transifex. I agree it's necessary one of them and is good for UReport use the locale with more translations (`sr-rs@latin`). We can disable the sr-latn and remove locale(`sr-latn`) option for the languages list, or if necessary remove the folder too.

Comment (norkans7) [reply ureport/settings_common.py]: @matmsa27 right you just keep one Serbian option and its folder of locale files

Comment (matmsa27) [reply ureport/settings_common.py]: Right @norkans7 , it's done!

**Pull request merged**